compiler/rustc_ast_pretty/src/pprust.rs

```diff
@@ -873,11 +873,9 @@ impl<'a> State<'a> {
     {
         self.rbox(0, b);
         let len = elts.len();
-        let mut i = 0;
-        for elt in elts {
+        for (i, elt) in elts.iter().enumerate() {
             self.maybe_print_comment(get_span(elt).hi());
             op(self, elt);
-            i += 1;
             if i < len {
                 self.s.word(",");
                 self.maybe_print_trailing_comment(get_span(elt), Some(get_span(&elts[i]).hi()));
```

Comment:
This is behaviour change I think. In the first loop, `i` in old code is 1, now it is 0.
However I don't know if the old behaviour is intentional or not.

Reply:
The old code add 1 to `i` here so it's the same.

Reply:
oh right! Thanks.